oak_remote_attestation/proto/v1/messages.proto

```diff
@@ -97,19 +99,15 @@ message GetPublicKeyResponse {
 }
 
 message InvokeRequest {
-  // Body of the request, encrypted with the enclave public key.
-  // TODO(#3442): Specify encryption format.
-  // TODO(#3442): Also pass client public or symmetric key to use for response encryption. Note that
-  // needs to be at least authenticated by the client, if not encrypted within this field.
-  // TODO(#3843): Use explicit `oak_crypto` messages.
-  bytes encrypted_body = 1;
+  // Body of the request, encrypted using Hybrid Public Key Encryption (HPKE).
+  // <https://www.rfc-editor.org/rfc/rfc9180.html>
+  oak.crypto.v1.EncryptedRequest encrypted_request = 1;
```

Comment:
So is this a breaking change in terms of proto compatibility? i.e. do we need to update binaries and protos and libraries at the same time?

Reply:
Ok, there is probably some way of doing it incrementally, but up to you. In the future it may be useful to do these changes without breaking compatibility though.

Reply:
It must always be possible to do non-breaking changes, according to the normal proto guidelines. in this case, you could add another field (e.g. with number 2) for the request, then update the libraries to use either field (or both fields together), then import those changes, then deprecate the original field. Makes sense? We cannot keep making breaking changes forever :)

Reply:
(no need to do it here, but I want to make sure we are on the same page, i.e. that currently we are doing breaking changes, but it's possible to do them in a non-breaking way instead, although more complex)

Reply:
Perhaps keep track of the Tag number at the top of the proto, continue to add proto fields, and mark old ones as deprecated when they are no longer of use.

Reply:
I'm confused -- is the old field used anywhere at all now?

Reply:
It's only used internally

Reply:
So I decided to not keep the original proto field and make it a breaking change. But before submitting I will make a test internal update to see that no tests are failing.